README.md

```diff
@@ -20,3 +20,7 @@ There is currently no official installer for Earth Enterprise Fusion & Server.
 
 ### Known Issues
 Known issues can be tracked via [wiki page](https://github.com/google/earthenterprise/wiki/Known-Issues), which is updated weekly.
+
+### Slack
+<img src="https://github.com/google/earthenterprise/blob/master/slack_rgb.png" width="100"> Join the OpenGEE [Slack channel](http://slack.opengee.org)! 
```

Comment:
I think you need to make a landing page first in order for the default slack badge to work.
If you want something static you could use 
```
[![Slack Status](https://img.shields.io/badge/chat-slack-ff69b4.svg)](https://opengee.slack.com)
```

Also, I don't think <img> tags are proper markdown.